src/Microsoft.AspNet.Mvc.Core/HttpResponseStreamWriter.cs

```diff
@@ -146,9 +146,9 @@ public override void Flush()
             FlushInternal(true, true);
         }
 
-        public override async Task FlushAsync()
+        public override Task FlushAsync()
         {
-            await FlushInternalAsync(true, true);
+            return FlushInternalAsync(true, true);
```

Comment:
While you're changing this line of code, could you also add named parameters to the boolean parameters?